packages/front-end/components/Features/FeaturesOverview.tsx

```diff
@@ -960,10 +962,18 @@ export default function FeaturesOverview({
                     </a>
                   </Box>
                 </Flex>
-                <Flex align="center" justify="end">
+                <Flex
+                  align={{ initial: "center", xs: "center", sm: "start" }}
+                  justify="end"
+                  flexShrink="0"
+                  direction={{ initial: "row", xs: "column", sm: "row" }}
+                  // @ts-expect-error text wrap is not valid for some reason, but it is
+                  style={{ textWrap: "nowrap" }}
```

Comment:
`text-wrap` is a [new CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/text-wrap), so maybe we need to update the TS version so it is recognized?

But if we just want to use `nowrap` maybe we should use `whiteSpace: nowrap` as it has broader support?

Reply:
whiteSpace nowrap works, good call.